### ISSUE TITLE
Balance

### ISSUE THREAD

**Issue by ca-lida** [mdbesten/what-could-go-wrong#1]
3 - Fluky

**Comment by B4dCow** [comment]
To fix balance, we can change the distribution of the czar card so it changes every turn, it fixes randomness issues that can affect the balance of the game

**Comment by ca-lida** [comment]
I think we might have messed up when playing the game, I don't think you're supposed to actually role the dice each time lol

**Comment by ca-lida** [comment]
Yeah we definitely messed up, i dont think balance is as much of an issue anymore, closing

**Comment by ca-lida** [comment]
Reopening issue balecause I was wrong - you do roll dice again to select the Czar - to be fixed by Nicolas on the DE community fork
